challenge.rb

```diff
@@ -6,7 +6,12 @@
 # and should return either `true` or `false` based on whether `age` is less than `minimum_age`.
 
 def is_of_age? age, minimum_age
-  # Replace this comment with your code.
+  if age < minimum_age
+    result = true
+  else
+    result = false
+  end
+  result
```

Comment:
Should have been _greater than_, not _less than_. However, the question was a little ambiguous, so I won't take points off. Your solution, though otherwise correct, is a little redundant. We could have written it as 
```
def is_of_age? age, minimum_age
  age >= minimum_age
end
```